Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -0,0 +1,1291 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Core.EventDatum.Input;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem.Handlers;
+using Microsoft.MixedReality.Toolkit.Core.Services;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX
+{
+    public class BoundingBox : MonoBehaviour, IMixedRealityPointerHandler, IMixedRealityGestureHandler, IMixedRealitySpatialInputHandler, IMixedRealitySourceStateHandler
+    {
+        #region Enums
+
+        private enum FlattenModeType
```

Comment:
a brief descriptive summary of these enums would be helpful.

Reply:
👍 done